### DISCUSSION TITLE
Memory leak fix needed on the next .NET 7 Service Release

### DISCUSSION THREAD

**Discussion by dcavallini** [dotnet/maui#16039] [Ideas]
I'm referring to this issue: https://github.com/dotnet/maui/issues/13520.
I've seen that the current bug has been fix for .NET 8 preview but i think the community needs a bugfix in the previus version too.
Speaking for myself i need this bugfix as soon as possible beacouse it ins't allowing my company to release a product. 
We can't wait the .NET 8 version to became stable (not on preview), as menontioned in many issues, many companies don't allow programmers to use preview for releases pourposes.
Concluding, the question is: is there any hope to see this bufgix on .NET 7?

**Comment by MartyIX** [comment]
(I believe you can check in the https://github.com/dotnet/maui/tree/release/7.0.2xx-sr7 branch whether it contains a fix for the issue you are looking for. Plus you can actually test that branch using https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md#advanced-scenarios. Not much but HTH.)

**Reply by dcavallini** [reply]
Hello @MartyIX thanks for answering. I've seen those links and I think they'r usufull but I'd like to know if there is any hope to have a solution for releasing my app or I must wait untill November to the actual fix in .NET 8?
I don't see any release planned for .NET 7. And I can't try a branch that is not in the service release, that equals like trying the preview IMO.
Thanks for the suggesting anyway, I'll give that a further look and hope MS provides something ASAP.

**Reply by MartyIX** [reply]
There is a tagged .NET 7 version https://github.com/dotnet/maui/tags and it should be released this week. 

Other than that, we can unfortunately only speculate.
